config/kafka.js

```diff
@@ -3,128 +3,113 @@
  * author : Aman Karki
  * created-date : 08-Sep-2020
  * Description : Kafka Configurations related information.
-*/
-
+ */
 
 //dependencies
-const kafka = require('kafka-node');
-const SUBMISSION_TOPIC = process.env.SUBMISSION_TOPIC;
-const CERTIFICATE_TOPIC = process.env.PROJECT_SUBMISSION_TOPIC;
-const USER_DELETE_TOPIC = process.env.USER_DELETE_TOPIC;
+const kafka = require('kafka-node')
+const SUBMISSION_TOPIC = process.env.SUBMISSION_TOPIC
+const CERTIFICATE_TOPIC = process.env.PROJECT_SUBMISSION_TOPIC
+const USER_DELETE_TOPIC = process.env.USER_DELETE_TOPIC
 const USER_DELETE_ON_OFF = process.env.USER_DELETE_ON_OFF
+const submissionsConsumer = require(GENERICS_FILES_PATH + '/kafka/consumers/submissions')
```

Comment:
@Prajwal17Tunerlabs When do we use submissionConsumer and projectCertificateConsumer?

Reply:
actually we are not using submissionsConsumer, we are only using projectCertificateConsumer, i added this line at the time when i was facing issues with kafka and was experimenting with kafka and rc-service, i have removed submissionConsumer now and have pushed the changes